src/lib/plugin.js

```diff
@@ -110,6 +120,7 @@ module.exports = class PluginVirtual extends EventEmitter2 {
     this.getInfo = () => JSON.parse(JSON.stringify(this._info))
     this.isConnected = () => this._connected
     this.getAccount = () => this._account
+    this.authorize = (authToken) => (authToken === this._authToken)
```

Comment:
If the function just returns a boolean I would call it something like `isAuthorized`